Sources/MountebankSwift/Models/Stub/Response/Is/Is.swift

```diff
@@ -1,20 +1,19 @@
 import Foundation
 
 // swiftlint:disable type_name
-/// The `Is` response type represents a canned response that you define.The response
-/// fields will be those defined with each protocol.
+/// A regular predefined response. Merges the specified response fields with ``Imposter``.`defaultResponse`
```

Comment:
Maybe remove the `defaultResponse`. at the end of line 4;

Reply:
But it really merges with the `defaultResponse`property of the imposter

Reply:
ahh, misread it because of the quotes :)